modules/cookie-script/config/COOKIE_SCRIPT_ID.ts

```diff
@@ -0,0 +1 @@
+export default 'YOUR COOKIE-SCRIPT ID'
```

Comment:
You can export default null and throw an error to force user to override this file. Check out modules/magento/config/MAGENTO_URL.ts